src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -70,7 +70,14 @@ private async Task<HttpResponseMessage> GetFromDevopsAsync(string request)
             int count = 0;
             while ((downloadResp.StatusCode == HttpStatusCode.TooManyRequests || downloadResp.StatusCode == HttpStatusCode.BadRequest) && count < 5)
             {
-                var retryAfter = (downloadResp.Headers.RetryAfter is null) ? "10" : downloadResp.Headers.RetryAfter.ToString();
+                _telemetryClient.TrackTrace($"X-RateLimit-Limit: {downloadResp.Headers.GetValues("X-RateLimit-Limit")}, X-RateLimit-Resource: {downloadResp.Headers.GetValues("X-RateLimit-Resource")}");
+
+                string retryAfter = "300"; // DevOps uses sliding 5 minuites window. At which point we are sure that Rate limit will be reset.
+                if (downloadResp.Headers.RetryAfter is not null)
```

Comment:
Please use curly braces here. It's very easy for single line if statements to result in bugs down the road if people think they are making changes within the if/else block but actually aren't.